doc/extensibility/IProjectTreeModifier.md

```diff
@@ -1,18 +1,37 @@
-`IProjectTreeModifier`
+`IProjectTreeModifier` and `IProjectTreeModifier2`
 ======================
 
 **[Item template:](project_item_templates.md)** Project Tree Modifier extension
 
 The `IProjectTreeModifier` interface can be exported into the 
-`ConfiguredProject` scope to apply cosmetic changes to a project tree 
-or one of its nodes. This allows you to, for example, set the icon on 
+`UnconfiguredProject` scope to apply cosmetic changes to a project tree 
```

Comment:
I see `IProjectTreeModifier` being imported at *both* scopes by the tree provider. Horrifying thought, as it may be, it does appear that tree modifiers can in fact be exported at the configured project scope. Is this then an accurate change? Or did you observe that it didn't work for your scenario and that exporting at the UnconfiguredProject scope worked more reliably?